src/tester.rs

```diff
@@ -431,6 +431,24 @@ fn check_names<'a>(cmd_pairs: &[(String, Command)], tests: &HashMap<String, Test
     }
 }
 
+fn write_ignored(test_name: &str, message: &str, single_threaded : bool) {
+    // Grab a lock on stderr so that we can avoid the possibility of lines blurring
+    // together in confusing ways.
+    let stderr = StandardStream::stderr(ColorChoice::Always);
+    let mut handle = stderr.lock();
+    if !single_threaded {
+        handle
+            .write_all(&format!("\ntest lang_tests::{} ... ", test_name).as_bytes())
+            .ok();
+    }
+    handle
+        .set_color(ColorSpec::new().set_fg(Some(Color::Yellow)))
+        .ok();
+    handle.write_all(b"ignored").ok();
```

Comment:
I know I was responsible for the original code here, but I think we can now simplify lines 447-449 to:

```rust
handle.write_all(format!("ignored ({})").as_bytes()).ok();
```

?

Reply:
We do this because `ignored` should be orange and the text inside should be the default output colour. If we simplify it to that one-liner and don't reset the colour after then it bleeds onto the next line.

Reply:
Aha, I knew my past self was not as stupid as my current self :p

Reply:
Aha, I knew my past self was not as stupid as my current self :p